source/unified-test-format/tests/Makefile

```diff
@@ -62,91 +60,91 @@ all: atlas-data-lake \
 
 connection-monitoring-and-pooling: connection-monitoring-and-pooling/logging
 
 connection-monitoring-and-pooling/logging: HAS_AJV
-	@ajv test -s $(SCHEMA) -d "../../connection-monitoring-and-pooling/tests/logging/*.yml" --valid
+	@./check-files.sh valid ../../connection-monitoring-and-pooling/tests/logging/*.yml
 
 crud: HAS_AJV
-	@ajv test -s $(SCHEMA) -d "../../crud/tests/unified/*.yml" --valid
+	@./check-files.sh valid ../../crud/tests/unified/*.yml
 
 gridfs: HAS_AJV
-	@ajv test -s $(SCHEMA) -d "../../gridfs/tests/*.yml" --valid
+	@./check-files.sh valid ../../gridfs/tests/*.yml
 
 index-management: HAS_AJV
-	@ajv test -s $(SCHEMA) -d "../../index-management/tests/*.yml" --valid
+	@./check-files.sh valid ../../index-management/tests/*.yml
 
 load-balancers: HAS_AJV
-	@ajv test -s $(SCHEMA) -d "../../load-balancers/tests/*.yml" --valid
+	@./check-files.sh valid ../../load-balancers/tests/*.yml
 
 read-write-concern: HAS_AJV
-	@ajv test -s $(SCHEMA) -d "../../read-write-concern/tests/operation/*.yml" --valid
+	@./check-files.sh valid ../../read-write-concern/tests/operation/*.yml
 
 retryable-reads: HAS_AJV
-	@ajv test -s $(SCHEMA) -d "../../retryable-reads/tests/unified/*.yml" --valid
+	@./check-files.sh valid ../../retryable-reads/tests/unified/*.yml
 
 retryable-writes: HAS_AJV
-	@ajv test -s $(SCHEMA) -d "../../retryable-writes/tests/unified/*.yml" --valid
+	@./check-files.sh valid ../../retryable-writes/tests/unified/*.yml
 
 run-command: HAS_AJV
-	@ajv test -s $(SCHEMA) -d "../../run-command/tests/unified/*.yml" --valid
+	@./check-files.sh valid ../../run-command/tests/unified/*.yml
 
 server-discovery-and-monitoring: HAS_AJV
-	@ajv test -s $(SCHEMA) -d "../../server-discovery-and-monitoring/tests/unified/*.yml" --valid
+	@./check-files.sh valid ../../server-discovery-and-monitoring/tests/unified/*.yml
 
 server-selection: server-selection/logging
 
 server-selection/logging: HAS_AJV
-	@ajv test -s $(SCHEMA) -d "../../server-selection/tests/logging/*.yml" --valid
+	@./check-files.sh valid ../../server-selection/tests/logging/*.yml
 
 sessions: HAS_AJV
-	@ajv test -s $(SCHEMA) -d "../../sessions/tests/*.yml" --valid
+	@./check-files.sh valid ../../sessions/tests/*.yml
 
 transactions-convenient-api: HAS_AJV
-	@ajv test -s $(SCHEMA) -d "../../transactions-convenient-api/tests/unified/*.yml" --valid
+	@./check-files.sh valid ../../transactions-convenient-api/tests/unified/*.yml
 
 transactions: HAS_AJV
-	@ajv test -s $(SCHEMA) -d "../../transactions/tests/unified/*.yml" --valid
+	@./check-files.sh valid ../../transactions/tests/unified/*.yml
 
 unified-test-format: unified-test-format/invalid unified-test-format/valid-fail unified-test-format/valid-pass
 
 unified-test-format/invalid: HAS_AJV
 	@# Redirect stdout to hide expected validation errors
-	@ajv test -s $(SCHEMA) -d "invalid/*.yml" --invalid > /dev/null && echo "invalid/*.yml passed test"
+	@./check-files.sh invalid invalid/*.yml > /dev/null && echo "invalid/*.yml passed test"
```

Comment:
This is out of scope for this change, but maintaining an inclusive list of all test subdirectories seems dangerous because it can easily miss new specs or test directory changes. Can we switch to excluding the invalid test files and including all other test files in `source/`?